README.md

```diff
@@ -1,12 +1,28 @@
-# PHPdokuwiki2findologic
+# DokuWiki Plugin: FINDOLOGIC XML Export
 
-[![Travis](https://api.travis-ci.org/TheKeymaster/phpdokuwiki2findologic.svg)](https://travis-ci.org/TheKeymaster/phpdokuwiki2findologic)
+[![Travis](https://travis-ci.org/findologic/dokuwiki-plugin-findologic-xml-export.svg?branch=master)](https://travis-ci.org/TheKeymaster/phpdokuwiki2findologic)
 
-This repository creates an XML export based on the dokuwiki data with the FINDOLOGIC XML scheme. It uses the library [libflexport](https://github.com/findologic/libflexport) to generate the XML.
+This DokuWiki plugin creates an XML based on the DokuWiki metadata with the FINDOLOGIC XML scheme.  
+It uses the library [findologic/libflexport](https://github.com/findologic/libflexport) to generate the XML.
 
-**Note: Tests are not yet complete. Use it on your own risk.**
+For any other dependencies please refer to the [composer.json](https://raw.githubusercontent.com/findologic/dokuwiki-plugin-findologic-xml-export/master/composer.json) file.
 
-Dependencies:
+### Install
 
- * [findologic/libflexport](https://github.com/findologic/libflexport) | v0.0.3
- * [sebastianbergmann/phpunit](https://github.com/sebastianbergmann/phpunit) | v5.7
\ No newline at end of file
+ 1. Download this repository
+ 2. Go into your DokuWiki folder and open the directory `/lib/plugins/`
+ 3. Create a folder in this directory named `findologicxmlexport` and copy the downloaded files in it
+ 
+### Usage
+
+ - Open your DokuWiki Admin page and there you will see the newly installed Plugin
+ 
+ ![https://github.com/findologic/dokuwiki-plugin-findologic-xml-export/raw/package_to_dokuwiki_plugin/examples/example_admin.png](https://github.com/findologic/dokuwiki-plugin-findologic-xml-export/raw/package_to_dokuwiki_plugin/examples/example_admin.png)
```

Comment:
The link is referring to a branch. Maybe change it to master already?

Reply:
TODO:

- Link refers to this branch. Edit it already for the master branch, so it doesn't needs another commit for this.